src/nvim/log.c

```diff
@@ -77,20 +76,15 @@ FILE *open_log_file(void)
     return stderr;
   }
 
-  // expand USR_LOG_FILE and open the file
+  // get log file location and open the file
   FILE *log_file;
   opening_log_file = true;
   {
-    static char expanded_log_file_path[MAXPATHL + 1];
-
-    expand_env((char_u *)USR_LOG_FILE, (char_u *)expanded_log_file_path,
-               MAXPATHL);
```

Comment:
Bah, I thought it was something more interesting for some reason :p I thought that might be useful when debugging?